osbs/build/build_request.py

```diff
@@ -50,7 +50,7 @@ def __init__(self, build_json_store):
         self._resource_limits = None
         self._openshift_required_version = parse_version('1.0.6')
         # For the koji "scratch" build type
-        self.scratch = False
+        self.scratch = None
```

Comment:
Oh, I get it, I think. Don't provide a value, so as to avoid overriding it if it is set elsewhere.